src/pages/kb/user-guide/querying/query-parameters.md

```diff
@@ -81,17 +73,27 @@ WHERE
 	AND table1.relevant_date <= '{{ myDate.end }}'
 ```
 
-Conveniently, the parameter value picker below the query window displays a unified widget. Date Range parameters behave exactly like Date parameters but are meant to save you time.
+{% callout info %}
 
-![](/assets/images/docs/gitbook/date-range-picker.png)
+Date parameters are Text parameters under the hood. So you should wrap them in single quotes (`'`) or whatever your database uses to declare strings. Although they behave like Text parameters, Dates are still safe for us in embeds and share dashboards.
```

Comment:
```suggestion
Date parameters are Text parameters under the hood. So you should wrap them in single quotes (`'`) or whatever your database uses to declare strings. Although they behave like Text parameters, Dates are still safe for use in embeds and share dashboards.
```

(`us` -> `use`)

Not sure I'm happy about this description. Maybe just mention that the date/time value is inserted as a text into the query?